### PR TITLE
Address detail loading component added

### PR DESCRIPTION
**Pull request by ifeelfishy**
New loading component added + styling

### CODE REVIEW

File: client/src/components/Loading/index.js [hunk 1]
```diff
@@ -14,7 +15,18 @@ const Loading = ({width, height}: Props) => (
     <LoadingComponent type="tail_spin" width={width} height={height} fill="#0062db" />
   </div>
 )
-
+export const EntityDetailLoading = () => (
+  <ListGroupItem className="list-row">
+    <div className="loadingsmall list-row-item">
+      <LoadingComponent type="tail_spin" width={15} height={15} fill="#0062db" />
```
Comment (borisvida) [inline client/src/components/Loading/index.js]: Wouldn't the color feel better if it were a constant?

File: client/src/components/Loading/index.js [hunk 2]
```diff
@@ -14,7 +15,18 @@ const Loading = ({width, height}: Props) => (
...
+    </div>
+  </ListGroupItem>
+)
+export const ModalLoading = () => (
```
Comment (borisvida) [inline client/src/components/Loading/index.js]: I am not a big fan of exporting multiple components from one file. But maybe that's just a matter of preference :man_shrugging:

File: client/src/components/Loading/index.js (hunk 1 above)
Comment (ifeelfishy) [reply client/src/components/Loading/index.js]: Color is now a constant

File: client/src/components/Loading/index.js (hunk 2 above)
Comment (ifeelfishy) [reply client/src/components/Loading/index.js]: I prefer it this way, since the components are very similar, but my preference is not very strong...

Comment (borisvida) [reply client/src/components/Loading/index.js]: Neither is mine, keep it as it is :smile:

**Comment by mirecmrozek** [comment]
Please merge because he does not have write access.

**Pull request merged**